examples/advanced_operations/add_dynamic_search_ads_campaign.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""This code example adds a Dynamic Search Ads Campaign.
+
+To get campaigns, run basic_operations/get_campaigns.py
```

Comment:
Could you please annotate this code example a bit to make things easier for the reader? See the .NET example here: https://github.com/googleads/googleads-dotnet-lib/blob/master/examples/AdWords/CSharp/v201809/AdvancedOperations/AddDynamicSearchAdsCampaign.cs

Reply:
Done